src/chrome/content/rules/CIMB-Singapore.xml

```diff
@@ -1,45 +1,21 @@
 <!--
-
-	Hosts may take a long time to respond.
-	
-	Other CIMB Group rulesets:
-		- CIMB-Malaysia.xml
-		- CIMB-Indonesia.xml
-		- CIMB-Thailand.xml
-		- CIMB-Cambodia.xml
-
-	Non-functional domain:
-		- (www.)cimbbank.com.sg		(Expired cert 12/04/15 09:29)
-
+	For other CIMB coverage, see CIMB.com.xml
 -->
-
 <ruleset name="CIMB Singapore">
 
-	<target host=    "cimbclicks.com.sg" />
+	<target host="cimbbank.com.sg" />
+	<target host="www.cimbbank.com.sg" />
+
+	<target host="cimbclicks.com.sg" />
 	<target host="www.cimbclicks.com.sg" />
-	<target host=    "bizchannel.cimb.com.sg" />
+
```

Comment:
Do `cimb.com.sg` and `www.cimb.com.sg` work for you?

Reply:
No, I've added them in the ruleset's ruleset.